custom_components/yahoofinance/const.py

```diff
@@ -165,6 +168,13 @@
     DATA_REGULAR_MARKET_TIME,
 ]
 
+# Keys of price type values, keep same order as TIME_DATA_KEYS
+PRICE_DATA_KEYS: Final = [
```

Comment:
Suggest combining this with TIME_DATA_KEYS as a dictionary. Then we don't have to worry about them being in sync.

The new block can do `for t, p in PRICE_TIME_DATA_KEYS.items():` and the block using those constants can use `.values():`